chainer/functions/pooling/roi_average_align_2d.py

```diff
@@ -253,8 +253,8 @@ def forward_gpu(self, inputs):
             T roi_end_w = bottom_rois[n * 4 + 3] * spatial_scale;
 
             // Force malformed ROIs to be 1x1
-            T roi_height = max(roi_end_h - roi_start_h, (T)1.);
-            T roi_width = max(roi_end_w - roi_start_w, (T)1.);
+            T roi_height = max((T)(roi_end_h - roi_start_h), (T)1.);
```

Comment:
Does it make sense to use `static_cast` here instead of a C-style cast, both for safety and consistency?